erpnext/stock/doctype/item/item.py

```diff
@@ -472,7 +577,7 @@ def validate_conversion_factor(self):
 
 			if d.uom and cstr(d.uom) == cstr(self.stock_uom) and flt(d.conversion_factor) != 1:
 				frappe.throw(
-					_("Conversion factor for default Unit of Measure must be 1 in row {0}").format(d.idx))
+					_("Conversion factor for default Unit of Measure must be 1"))
```

Comment:
Why change to a less useful error message?

Reply:
Because the user will no longer use the old Conversion Factors table. Except they will be using the new Conversion Graph table and its row # cannot be determined (since the values may be computed using multiple rows from the conversion graph).

When this error is thrown, the (old) conversion factor table will not be updated and hence the row # will only confuse the user.